vms/platformvm/network/network.go

```diff
@@ -155,7 +145,16 @@ func (n *network) issueTx(tx *txs.Tx) error {
 	}
 
 	// Verify the tx at the currently preferred state
-	if err := n.manager.VerifyTx(tx); err != nil {
+	//
+	// We need to grab the context lock here to avoid racy behavior with
+	// transaction verification + mempool modifications.
+	//
+	// Invariant: tx should not be referenced again without the context lock
+	// held to avoid any data races.
```

Comment:
We seem to reference the tx again later in this function without the lock held

Reply:
See earlier response